testutil/runbook.go

```diff
@@ -0,0 +1,71 @@
+package testutil
+
+import (
+	"fmt"
+	"html/template"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+// BenchmarkSet returns a server and a path pattern for benchmarking.
+func BenchmarkSet(t testing.TB, bookCount, stepCount int, body string) (*httptest.Server, string) {
+	const httpRunbookTmpl = `desc: Test using HTTP
+runners:
+  req: {{ .Endpoint }}
+steps:
+{{ range $k := .Steps }}
+-
+  req:
+    /:
+      post:
+        body:
+          application/json:
+            {{ $.Body }}
+{{ end }}
```

Comment:
We want to increase coverage using as many different runners as possible.
test, bind, if, dump, eval, exec, loop etc..

Reply:
The benchmarks for this are based on the number of runbooks, number of steps, and request/response body size = store size, and I believe that runner differences are out of scope.